aten/src/ATen/native/native_functions.yaml

```diff
@@ -81,7 +81,7 @@
   variants: method
 
 - func: requires_grad_(Tensor(a!) self, bool requires_grad=True) -> Tensor(a!)
-  manual_kernel_registration: True
+  manual_cpp_binding: True
```

Comment:
Why does this need manual binding?

Reply:
I put them in manual binding mainly because these functions are only manipulating autograd related attributes on tensor instead of doing real operation with the tensor data. So I feel we could take some perf gain by moving them to manual binding :D